hpx/runtime/actions/continuation.hpp

```diff
@@ -124,12 +124,12 @@ namespace hpx
         }
     }
 
-    template <typename T>
-    void set_lco_value(naming::id_type const& id, naming::address && addr, T && t,
+    template <typename Result, typename RemoteResult>
+    void set_lco_value(naming::id_type const& id, naming::address && addr, RemoteResult && t,
```

Comment:
The problem here is that get_local_result and the local result of the typed
continuation might differ if we use something that transforms the remote
result with get_remote_result to a completely different type. One example
for this is `agas::response`.